SlideServer.py

```diff
@@ -6,13 +6,24 @@
 import string
 import sys
 import pyvips
+from os import listdir
+from os.path import isfile, join
 
 import flask
 import flask_cors
 import openslide
 from werkzeug.utils import secure_filename
 import dev_utils
 
+from flask import Flask, request, jsonify, _request_ctx_stack
+from flask_cors import cross_origin
+
+AUTH0_DOMAIN = 'YOUR_DOMAIN'
+API_AUDIENCE = False
+ALGORITHMS = ["RS256"]
+
+
```

Comment:
```suggestion
```
I don't think we're using any of this, right?

Reply:
Yeah, sorry missed it